index.html

```diff
@@ -2435,37 +2435,58 @@ <h2>Cryptographic Suites</h2>
 verification of the data.
         </li>
         <li>
-          The specification MUST define a <dfn>suite selection algorithm</dfn> that accepts a [=JSON
-          object=] |proof| and uses the properties described in [[[#proofs]]] to return either
-          failure or a [=VC cryptographic suite=] [[Infra]] [=struct=]. This algorithm SHOULD be
-          registered in [[[VC-SPECS]]]. A
-          <dfn class="export">VC cryptographic suite</dfn> [=struct=] has the following
-          [=struct/items=].
+The specification MUST define a <dfn>suite selection algorithm</dfn> that
+accepts a set of options ([=map=] |options|) and returns a
+[=cryptographic suite=] ([=struct=] |cryptosuite|). This algorithm SHOULD be
+registered in [[[VC-SPECS]]]. A cryptographic suite [=struct=] has the
+following [=struct/items=].
           <dl data-dfn-for="cryptographic suite">
             <dt><dfn>transform</dfn></dt>
-            <dd>A [=transformation algorithm=] that takes a [=JSON object=] as input and returns a
-            [=JSON object=].</dd>
+            <dd>
+A [=transformation algorithm=] that takes an [=unsecured data document=]
+([=JSON object=] |unsecuredDocument|) and transformation options
+([=struct=] |options|) as input and produces a
```

Comment:
"struct" shouldn't really be used as the type of an input parameter, any more than you use "struct" as the type of a C variable. Instead, you create concrete types as structs of a particular set of "items", and then input parameters are typed by those concrete types. If you want an unbounded set of keys, use a [map](https://infra.spec.whatwg.org/#maps).

Reply:
Fixed to use a map.